code/game/objects/items/stacks/tickets.dm

```diff
@@ -29,4 +29,18 @@
 		qdel(src)
 
 /obj/item/stack/arcadeticket/thirty
-	amount = 30
\ No newline at end of file
+	amount = 30
+
+/obj/machinery/computer/arcade/attackby(obj/item/O, mob/user, params)
+	if(istype(O, /obj/item/stack/arcadeticket))
+		var/obj/item/stack/arcadeticket/T = O
+		var/amount = T.get_amount()
+		if(amount <2)
+			to_chat(user, "<span class='warning'>You need 2 tickets to claim a prize!</span>")
+			return
+		prizevend(user)
+		T.pay_tickets()
+		T.update_icon()
+		O = T
+		to_chat(user, "<span class='notice'>You turn in 2 tickets to the [src] and claim a prize!</span>")
+		return
```

Comment:
This either needs to be ported upstream to Polaris, or given `//CHOMPStation edit` tags.